test/README.md

```diff
@@ -53,8 +53,27 @@ AMQP URL
 
     export RACKHD_AMQP_URL=[amqp://<url>:<port> default=amqp://localhost:9091]
 
-Specify test groups
+OS Installer locations
+
+    Specify specific OS repository path:
+
+    export RACKHD_CENTOS_REPO_PATH=[location to http mirror for all CentOS images]
+    export RACKHD_ESXI_REPO_PATH=[location to http mirror for all ESXi images]
+    export RACKHD_UBUNTU_REPO_PATH=[location to http mirror for all Ubuntu images]
+    export RACKHD_SUSE_REPO_PATH=[location to http mirror for all SUSE images]
+
+    Example: export RACKHD_CENTOS_REPO_PATH=http://ip:port/repo/centos
+
+HTTP proxy configuration (optional):
+
+To define HTTP proxies for accessing remote OS image repositories, see the [RackHD Configuration howto](http://rackhd.readthedocs.io/en/latest/rackhd/configuration.html?highlight=httpProxies#rackhd-configuration)
```

Comment:
The httpProxies doc is only a general description, I suggest giving some concrete example for each OS here or in httpProxies doc, because I realized most people doesn't understand how to set a correct repo address.